src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -76,6 +79,54 @@ public String getOgcCollection(@RequestParam(value = "f", required = false, defa
 		return rtn;
 	}
 
+	@Operation(
+			description = "Return GeoJSON Data specific to the features in the requested Collection.",
+			responses = {
+					@ApiResponse(
+							responseCode = "200",
+							description = "GeoJSON representation of the Feature.",
+							content = @Content(schema = @Schema(implementation = FeatureGeoJSON.class))),
+					@ApiResponse(
+							responseCode = "404",
+							description = "The requested collection feature was not found.",
+							content=@Content())
+			},
+			externalDocs=@ExternalDocumentation(url="http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_feature_")
+			)
+	@GetMapping(value = "collections/{collectionId}/items", produces = MediaType.APPLICATION_JSON_VALUE)
+	public String getOgcCollectionFeatures(
+			@RequestParam(value = "f", required = false, defaultValue = "json") String mimeType,
+			@RequestParam(value = "limit", required = false, defaultValue = "${query.limit.max}") Integer limit,
+			@RequestParam(value = "startIndex", required = false, defaultValue = "0") int startIndex,
+			@RequestParam(value = "bbox", required = false) List<Double> bbox,
+			@PathVariable(value = "collectionId") String collectionId, HttpServletResponse response) {
+
+		String rtn = getOgcCollection(mimeType, collectionId, response);
+		if (response.getStatus() == HttpServletResponse.SC_OK) {
+			Map<String, Object> params = collectionsParams.getParameters(collectionId);
+			int count = collectionsDao.getCollectionFeatureCount(params);
+
+			Integer limitParam = limit;
+			if (limitParam == null) {
```

Comment:
is this limitParam logic redundant with the code in CollectionParams?
same with the other paging related parameters.